pkg/log/messages/error_messages.go

```diff
@@ -20,7 +20,7 @@ const CSPFK003E string = "CSPFK003E AccessToken failed to delete access token da
 // Environment variables
 const CSPFK004E string = "CSPFK004E Environment variable '%s' must be provided"
 const CSPFK005E string = "CSPFK005E Provided incorrect value for environment variable %s"
-const CSPFK007E string = "CSPFK007E Setting SECRETS_DESTINATION environment variable to 'k8s_secrets' must run as init container"
+const CSPFK007E string = "CSPFK007E Setting SECRETS_DESTINATION environment variable to 'k8s_secrets' must run as init or application container"
```

Comment:
i would change this to `must not run as a sidecar container`

Reply:
ideally the user can put
```
- name: CONTAINER_MODE
   value: sigal
```
So I want to put feedback on the types of input we do accept

Reply:
would love to know what the `sigal` container mode gives you!

but i really think that we should say what we do not allow as that is the issue here

Reply:
I disagree with you here. Because we don't know if they inputted "sidecar". It could ideally be anything and we are assuming sidecar. IMO it is better to guide the user to tell them what we do support

Reply:
> would love to know what the sigal container mode gives you!

LOTS of avocados

Reply:
![image](https://user-images.githubusercontent.com/29940926/90110882-8b2b0600-dd56-11ea-8345-cfd8285c1341.png)
`container_mode=sigal`, illustration

Reply:
which one is the sidecar?

Reply:
Such a philosophical question because the avocados in the plastic box are totally containers and the box is the pod. SO I would say the water surrounding the cados is the sidecar (why is there even water in the plastic bin..?)